sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -1898,9 +1889,9 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
             {
               path.emplace_back(XmlTagName::c_StorageServiceProperties);
             }
-            else if (std::strcmp(node.Name, "Protocol") == 0)
+            else if (std::strcmp(node.Name, "ProtocolSettings") == 0)
             {
-              path.emplace_back(XmlTagName::c_Protocol);
+              path.emplace_back(XmlTagName::c_ProtocolSettings);
             }
             else if (std::strcmp(node.Name, "Cors") == 0)
```

Comment:
```suggestion
            else if (std::strcmp(node.Name, XmlTagName::c_Cors) == 0)
```

Reply:
`XmlTagName::c_Cors` is the enum used to identify the current level of XML.